src/utils/index.js

```diff
@@ -70,11 +70,14 @@ export const getTimeSince = time => {
 };
 
 /**
- * Convert bytes to gigabytes and return a nicely formatted number.
+ * Convert bytes to MB, GB, etc. and return a nicely formatted number.
  *
- * @param bytes
+ * @param {Number} bytes
  * @returns {string}
  */
-export const convertBytesToGigabytes = bytes => {
-	return Number( bytes / 1073741824 ).toLocaleString( undefined, { maximumFractionDigits: 0 } );
+export const getHumanReadableBytes = bytes => {
```

Comment:
I had a look and there was one unused reference to the old function name in a file that's not currently imported. I'll fix that one up. Just something to double check - I'm happy for you to patch things up as and where you find problems if you have time.